searx/help/en/search-syntax.md

```diff
@@ -0,0 +1,35 @@
+# Search syntax
+
+SearXNG allows you to modify the default categories, engines and search language
+via the search query.
+
+Prefix `!` to set category and engine names.
+
+Prefix: `:` to set the language.
+
+Abbrevations of the engines and languages are also accepted.  Engine/category
+modifiers are chainable and inclusive.  E.g. with {{search('!map !ddg !wp paris')}}
+search in map category **and** duckduckgo **and** wikipedia for
+`paris`.
+
+See the {{link('preferences', 'preferences')}} for the list of engines,
```

Comment:
That is true / I did not had an idea how to make it smarter for translators.

For search links we have:

    {{search('!wp paris')}}

Instead of:

    {{link('preferences', 'preferences')}}

we can use:

    {{link('preferences --> preferences')}}

but I do not know if it is better / any other suggestion for a markup of links that needed to be rendered in a Flask context?